src/python/espressomd/system.pxd

```diff
@@ -34,3 +36,13 @@ cdef bool skin_set
 cdef extern from "particle_data.hpp":
     int init_type_map(int type) except +
     int number_of_particles_with_type(int type) except +
+
+cdef extern from "PartCfg.hpp":
```

Comment:
why did you put the particle stuff here?

Reply:
`dist_to` takes a `partCfg()` as first argument, so we have to declare it in the .pxd file

Reply:
but why does it belong to ```system```?

Reply:
it doesn't, it's only a side-effect of the underlying core function; we could remove that partCfg dependency by writing an overloaded `dist_to` function in the core

Reply:
as mentioned below, I think we don't need the function at all... I think we should just have a free function that does the minimum image convention for us taking a position and the box as an input a.k.a. ```get_mi_vector```